src/vue/forms/LoginForm.vue

```diff
@@ -25,6 +25,21 @@
         />
       </div>
     </div>
+    <div
+      v-if="tfaError"
+      class="app__form-row"
+    >
+      <div class="app__form-field">
+        <input-field
+          v-model="form.tfaCode"
+          @blur="touchField('form.tfaCode')"
+          id="login-tfa-code"
```

Comment:
why do we need an id here?

Reply:
https://github.com/tokend/web-client/pull/38#discussion_r253804830